### PR TITLE
Improved formatting of trigsimp docstring.

### PR DESCRIPTION
**Pull request by moorepants**
#### Brief description of what is fixed or changed

Improved formatting of trigsimp docstring.

New:
image

Old:
image

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/simplify/trigsimp.py [hunk 1]
```diff
@@ -427,23 +427,32 @@ def analyse_gens(gens, hints):
...
+        - ``'old'``
+
+        If ``'matching'``, simplify the expression recursively by targeting
+        common patterns. If ``'groebner'``, apply an experimental groebner
+        basis algorithm. In this case further options are forwarded to
+        ``trigsimp_groebner``, please refer to its docstring. If
+        ``'combined'``, it first runs the groebner basis algorithm with small
+        default parameters, then runs the ``'matching'`` algorithm. If ``'fu'``
+        runs the collection of trigonometric transformations described by Fu,
+        et al. (see the ``fu`` docstring). If ``'old'`` the original SymPy trig
```
Comment (oscargus) [inline sympy/simplify/trigsimp.py:451]: Is it possible to link to the fu method? Other than that LGTM!

Comment (moorepants) [reply sympy/simplify/trigsimp.py]: done

File: sympy/simplify/trigsimp.py [hunk 2]
```diff
@@ -427,23 +427,33 @@ def analyse_gens(gens, hints):
...
+        If ``'matching'``, simplify the expression recursively by targeting
+        common patterns. If ``'groebner'``, apply an experimental groebner
+        basis algorithm. In this case further options are forwarded to
+        :py:func:`~sympy.simplify.trigsimp.trigsimp_groebner`, please refer to
+        its docstring. If ``'combined'``, it first runs the groebner basis
+        algorithm with small default parameters, then runs the ``'matching'``
+        algorithm. If ``'fu'`` runs the collection of trigonometric
+        transformations described by Fu, et al. (see the
+        :py:func:`~sympy.simplify.fu.fu` docstring). If ``'old'`` the original
+        SymPy trig simplication function is run.
```
Comment (smichr) [inline sympy/simplify/trigsimp.py:453]:
```suggestion
        algorithm. If ``'fu'``, run the collection of trigonometric
        transformations described by Fu, et al. (see the
        :py:func:`~sympy.simplify.fu.fu` docstring). If ``'old'``, the original
        SymPy trig simplication function is run.
```

**Comment by smichr** [comment]
```
/home/runner/work/sympy/sympy/sympy/simplify/trigsimp.py:docstring of sympy.simplify.trigsimp.trigsimp:18: WARNING: py:func reference target not found: sympy.simplify.trigsimp.trigsimp_groebner
```

**Comment by moorepants** [comment]
Doesn't seem like `trig_groeber` is documented.

File: sympy/simplify/trigsimp.py [hunk 3]
```diff
@@ -427,23 +427,33 @@ def analyse_gens(gens, hints):
...
+        - ``'matching'``, default
+        - ``'groebner'``
+        - ``'combined'``
+        - ``'fu'``
+        - ``'old'``
+
+        If ``'matching'``, simplify the expression recursively by targeting
+        common patterns. If ``'groebner'``, apply an experimental groebner
+        basis algorithm. In this case further options are forwarded to
+        :py:func:`~sympy.simplify.trigsimp.trigsimp_groebner`, please refer to
```
Comment (moorepants) [inline sympy/simplify/trigsimp.py:447]:
```suggestion
        ``trigsimp_groebner``, please refer to
```

**Comment by moorepants** [comment]
I've addressed everything and tests pass. @smichr or @oscargus can you press merge if you are good with this?

**Pull request merged**